OptimizelySDK/Optimizely.cs

```diff
@@ -440,7 +443,26 @@ public Variation GetForcedVariation(string experimentKey, string userId)
             return DecisionService.GetForcedVariation(experimentKey, userId, config).ResultObject;
         }
 
-        #region  FeatureFlag APIs
+        public Variation GetFlagVariationByKey(string flagKey, string variationKey)
+        {
+            var flagVariationMap = ProjectConfigManager?.GetConfig().FlagVariationMap;
```

Comment:
Don't work directly on flagVariationMap, store config in a variable, check IsValid and move forward.